index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
I did ask for exactly this, thanks. At the end of the day, I want to have a site that I myself can maintain in the worst case scenario, and I am by no means an experienced web UX developer (or actually I am - but pretty much all of my experience is pre 2005...). 

Brook, you did a fantastic job putting this together - but when you left, there was no one who understood it well enough to continue with it. I don't want to end up there again.